giraffe/package.json

```diff
@@ -89,5 +89,16 @@
   "peerDependencies": {
     "react": "^16.8.0",
     "react-dom": "^16.8.0"
+  },
+  "dependencies": {
+    "leaflet": "^1.6.0",
```

Comment:
These dependencies shouldn't be here. They should be in giraffe/giraffe/package.json under `devDependencies`. 

The reason for the nested folder structure is because Giraffe has a separate folder for the demo page. This keeps Giraffe compact. And we can keep these in `devDependencies` because when we publish a new version, the dependencies will be found correctly and rolled into the artifact.

Reply:
Just to be clear, **_all_** of the added dependencies should be moved, not just `leaflet`. I clicked the comment at the wrong line.

Reply:
Sure, I will do that.

Reply:
It is done.